.github/workflows/vite-pull.yaml

```
@@ -0,0 +1,26 @@
+name: vite-pull
+on:
+  pull_request:
+    paths:
+      - vite/**
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          sparse-checkout: vite
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
+          registry-url: https://registry.npmjs.org
+      - name: Install winglang
+        run: npm i -g winglang
+      - name: Install dependencies
+        run: npm install
+        working-directory: vite
+      - name: Test
+        run: wing test **/*.test.w
```

This isn't needed, `wing test` will grab all tests in the directory for you

```suggestion
        run: wing test
```

---

Thanks! I didn't think of changing the script too much.